### ISSUE TITLE
Snowflake identity for digital assets

### ISSUE THREAD

**Issue by sadfool1** [HydroBlockchain/Community-Brainstorming#34]
ERC 1484 or snowflake creates a digital identity of dApps. I viewed briefly the example of the PetFriendResolver and I wonder if you can give a unique ID to digital assets like steam game items.

Suppose I create an offer to buy this specific 'sword' for say 1000 Hydro, anyone who's willing to sell me can do so with a simple resolving algorithm (Opskins is a centralized marketplace in which gamers can buy/sell their virtual items for [WAX](https://wax.io/)), so I believe we can include these digital assets into the Hydro dAppstore. Unless it's limited to tangible assets.

I believe Steam has their own way of creating digital fingerprint on their items (because i have tried researching how to clone these items) so I think we can seek them for advice if we want to create an ecosystem of a virtual products marketplace within the Hydro platform. However, this proposal is not limited to only Steam game items but to the broader virtual marketplace from Fortnite, WoW to Overwatch.

**Comment by AndyHydro** [comment]
I think this could be pretty cool. Ideally this would be a resolver which has a marketplace of ERC721s.

**Comment by sadfool1** [comment]
Yeah the digital goods like those on OPS skins are like ERC721 similar to cryptokitties. So does this mean that virtual goods such as these cannot be traded around snowflake? 

If not, will we be able to fit this into other Hydro protocols?

**Comment by jzafrap** [comment]
FYI, there is a very good example of ERC721 here: [pixelCons ](https://github.com/PixelCons), that I liked, and is open source with sources on github, if you want to give a look.
